mysite/base/decorators.py

```diff
@@ -213,3 +214,17 @@ def get(self):
             return x
 
     return property(get)
+
+def authenticated(func):
+    """ decorator that redirect user to next page if
+    he is already logged."""
```

Comment:
Typically, we try to keep documentation gender-neutral. Perhaps change "he" to "the user"